spyderlib/spyder.py

```diff
@@ -1928,6 +1940,56 @@ def quick_layout_switch(self, index):
             action = plugin.toggle_view_action
             action.setChecked(plugin.dockwidget.isVisible())
 
+    # --- Other
+    def save_visible_toolbars(self):
+        """ """
+        print('save')
+        temp = []
+        for toolbar in self.visible_toolbars:
+            temp.append(toolbar.objectName())
+        CONF.set('main', 'last_visible_toolbars', temp)
+
+    def load_last_visible_toolbars(self):
+        """ """
+        print('load')
+        toolbars_names = CONF.get('main', 'last_visible_toolbars')

```

Comment:
You need to use here the `default` option of `CONF.get`, like this

```python
CONF.get('main', 'last_visible_toolbars', default=[])
```

given that it's not going to be part of `config.py`